### DISCUSSION TITLE
使用有道翻译引擎翻译文档注释 提示 [翻译文档失败: 缺少必需参数]

### DISCUSSION THREAD

**Discussion by Radioactive821** [YiiGuxing/TranslationPlugin#4810] [Q&A] [answered]
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/64411954/10eac8be-4016-45f3-a2a2-d4e81fabc983)
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/64411954/eaf1d946-6571-451f-ab1d-4595807d2b93)
试了很多次都是翻译失败

**Comment by YiiGuxing** [comment, marked as answer]
请更新到3.5.6，已在此版本中修复。

**Reply by Radioactive821** [reply]
好嘞，感谢大佬
